EuroPythonBot/extensions/job_board/Job Posts 2024 (Antworten) - Formularantworten 2.csv

```diff
@@ -0,0 +1,294 @@
+Zeitstempel,E-Mail-Adresse,Job title,Company name,Job Location,Deadline,Job description,Duties and responsibilities,Appreciated qualification and experience,Additional info (optional),Contact person name,Contact person email,URL/Link to job posting,Picture for job posting (if available)
```

Comment:
[minor] this file name is very German ^^'

Reply:
renamed the file :)